dotcom-rendering/src/model/enhanceCards.ts

```diff
@@ -75,6 +77,38 @@ const enhanceSupportingContent = (
 	});
 };
 
+const decideAvatarUrl = (
+	tags?: TagType[],
+	byline?: string,
+): string | undefined => {
+	if (!tags || tags.length === 0) return;
+	const soleContributor = getSoleContributor(tags, byline);
+	// eslint-disable-next-line consistent-return -- because I want to return undefined and you can't stop me
```

Comment:
The problem is that you’re returning inconsistently two lines above, which I don’t think you need to do.

```suggestion
	const soleContributor = getSoleContributor(tags, byline);
	// eslint-disable-next-line consistent-return -- because I want to return undefined and you can't stop me
```

Reply:
It still needs the check for `tags` but it seems ts doesn't recognise `return` as being the same as `return undefined`, so I helped it out by saying that

Reply:
You could forgo checking for tags by setting a default in where you’re calling `getSoleContributor` like so:

```ts
getSoleContributor(tags ?? [], byline);
```

Reply:
Added a default